ngraph/core/reference/include/ngraph/runtime/reference/matmul.hpp

```diff
@@ -146,13 +184,8 @@ namespace ngraph
                 // Inputs are 2D and below, perform dot directly
                 if (arg0_rank <= 2 && arg1_rank <= 2)
                 {
-                    dot(arg0_update,
```

Comment:
Do we still need to have `dot` method?

Reply:
Actually, I didn't find any other usage of this old `dot` reference so we can remove it.

Reply:
Done